lib/services/local/access_list.go

```diff
@@ -670,10 +670,6 @@ func (a *AccessListService) VerifyAccessListCreateLimit(ctx context.Context, tar
 		return trace.Wrap(err)
 	}
 
-	if len(lists) == 0 {
```

Comment:
I'm torn on deleting this, but I think its probably better to have one source of truth for our AccessList limit (the default AccessList entitlement) rather than have it also encoded here as a side-effect.

Reply:
Retored (with a `//TODO` to remove)